tawk_to/src/core/TawktoGenerator.php

```diff
@@ -65,6 +64,69 @@ public function getWidget()
         return $output;
     }
 
+    private function shouldDisplayWidget($options = null)
```

Comment:
The logic here was gotten from the removed lines of [tawk_to/tawk_to.module lines 48 to 125](https://github.com/tawk/tawk-drupal8/pull/3/files#diff-10292e47e00261b543b447e84bb9bc31e6274c56a69339798317053b1a8860d6L48)

Reply:
Transferred it here in the generator file so that when the cache gets invalidated, it goes through this logic again.